x-pack/plugins/log_explorer/public/components/flyout_detail/sub_components/highlight_container.tsx

```diff
@@ -33,9 +33,7 @@ export const HighlightContainer = React.forwardRef<HTMLDivElement, HighlightCont
     return shouldRender ? (
       <div ref={ref}>
         <EuiHorizontalRule margin="xs" />
-        <EuiPanel paddingSize="m" hasShadow={false} hasBorder={true}>
-          {flexChildren}
-        </EuiPanel>
+        {flexChildren}
```

Comment:
**nit:** now that the container is gone, per design we are making more compact also spacing after this whole section, I think we should also update the spacing between the three main sections, as the AI Assistant look distant from the highlights compared to the design.

Reply:
will go with `gutterSize:m`